posts/inside-rust/2022-04-13-imposter-syndrome.md

```diff
@@ -0,0 +1,91 @@
+feeling that we aren't as qualified as our peers. This makes us feel
+unqualified and undeserving compared to those around us. It can make us
+hesitate to join communities in the first place and, for those already
+involved, it can create a sense of impending doom where you constantly feel
+like you're going to get found out and expelled from the community. Overall
+it's just not great for mental health, 0/10, would not recommend.
+
+The thing is though, imposter syndrome is a logical fallacy[^2]. Imposter
+syndrome occurs when we discount what we know and inflate what we think other
+people know, and this effect is often then reinforced by systemic bias for
+those of us who don't get the assumption of competence.
+
+![picture of imposter syndrome, left side shows a large circle saying "What I think others know" and a small circle inside of it saying "What I know", right side shows the same small circle saying "What I know" surrounded by many other equally sized small circles labeled "What others know"](/static/images/2022-04-13-imposter-syndrome/imposter_syndrome.jpg)
+
+In reality, we're all specialists within the Rust project. We all have areas
+where we have deep expertise and other large areas where we only have the
+vaguest idea of how things work. Niko still comes to me to ask questions about
+error handling. I frequently need to tell my fellow contributors that I have no
+idea what the acronyms they're using mean[^3]. But this doesn't mean we don't
+deserve our positions within the project. We don't expect every contributor
+know everything, to be perfect, or to make no mistakes. The only thing we
+expect from our contributors is the ability to collaborate effectively with
+others and the ability to learn and grow over time.
+
+The thing that makes the Rust project as good as it is isn't a couple of
+prolific contributors lifting mountains by themselves, it's everyone working
+together that brought us to where we are today. We all make mistakes. The
+project has layer[^4] after layer[^5] of safeguards to make sure we have a
+chance to catch and fix them before they affect our users. These incidents are
+unavoidable, expected, and honestly fine! We want people to feel empowered to
+make changes they're not 100% confident in, to make mistakes, to learn, and to
+grow within the Rust project. This is how all of us got to where we are today!
+
+So, if you look up to people within the Rust project, if the work we do here
+interests you, if you have always wanted to contribute, and _especially_ if you
+already have contributed, I want you to know that you're one of the people we
+want to apply for [Rust Foundation grants and
+fellowships](https://foundation.rust-lang.org/grants/). You're one of the
+people we want to eventually see join teams. You're not an imposter, and I want
+you to know that I really look forward to seeing you around the project.
+
+---
+
+*To help reinforce and normalize this, I want to turn this end of this blog
```

Comment:
I like this idea but I wonder if there's a better spot to put this? I think reviewing might get overwhelming and eventually folks will not look at this blogpost. I think that maybe a microsite for the grants program could be a great location for this content!

Reply:
Yes, that's a great idea. I have so many cool anecdotes that belong here!